### DISCUSSION TITLE
How to supply ASR evaluation cleaner file?

### DISCUSSION THREAD

**Discussion by feddybear** [espnet/espnet#3632] [Q&A] [answered]
As a follow up to #3625 , I now wish to know how to supply the "--cleaner" file for run.sh on the ASR recipes.  Upon searching the documentation, I only found available options for TTS. For Japanese ASR, for example, I would like to do some preprocessing like half2wide or hira2kana conversion, what are the accepted formatting for the cleaner file?

**Comment by sw005320** [comment, marked as answer]
I usually perform such processing in the data preparation (`local/data.sh`).
Usually, this is enough.
You can check https://github.com/espnet/espnet/blob/master/egs2/csj/asr1/local/data.sh and related files.

**Reply by feddybear** [reply]
Again, thank you!

**Reply by feddybear** [reply]
@sw005320 Watanabe-san, sorry for asking this here because I don't know where to ask, but are there any chance that pretrained kaldi TDNN-F models on the LaboroTV dataset can be made publicly available (similar to ESPnet2's CTC-Transformer pretrained model)?

**Reply by sw005320** [reply]
@hfujihara would have a model.
